src/routes/web.php

```diff
@@ -3,14 +3,32 @@
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\PostController;
 use App\Http\Controllers\UserController;
-use App\Http\Controllers\LoginController;
-use App\Http\Controllers\RegisterController;
+use App\Http\Controllers\Auth\LoginController;
+use App\Http\Controllers\Auth\RegisterController;
+use App\Http\Controllers\Auth\VerificationController;
+use App\Http\Controllers\Auth\ResetPasswordController;
+use App\Http\Controllers\Auth\ForgotPasswordController;
 
-Route::get('/', [LoginController::class, 'index'])->name('view.login');
-Route::post('/index', [LoginController::class, 'auth'])->name('login.auth');
+Route::middleware(['auth', 'verified'])->group(function () {
+    Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
+    Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
+    Route::get('/home', [UserController::class, 'home'])->name('user.home');
+    Route::post('/posts', [PostController::class, 'store'])->name('post.store');
+});
+
+Route::get('/', [LoginController::class, 'index'])->name('login');
+Route::post('/login', [LoginController::class, 'auth'])->name('login.auth');
```

Comment:
Why still no middleware? Research about Guest Middleware in laravel!

Reply:
I already resolved this Sir according to your comment, thank you!